pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -1,45 +1,61 @@
 ---
 permalink: /documentation/getting-started/developers/phase-two-compile/
 layout: styleguide
-title: "Phase 2: Compile"
+title: 'Phase 2: Compile'
 category: How to use USWDS
 lead: Compile Design System source code into browser-readable files.
 type: docs
 subnav:
-  - text: "Step 1: Set up your project’s Sass entry point"
-    href: "#step-1-set-up-your-projects-sass-entry-point"
-  - text: "Step 2: Install uswds-compile"
-    href: "#step-2-install-uswds-compile"
-  - text: "Step 3: Create a gulpfile"
-    href: "#step-3-create-a-gulpfile"
-  - text: "Step 4: Create path settings and export compile functions"
-    href: "#step-4-create-path-settings-and-export-compile-functions"
-  - text: "Step 5: Initialize your project"
-    href: "#step-5-initialize-your-project"
-  - text: "Step 6: Verify successful installation"
-    href: "#step-6-verify-successful-installation"
+  - text: 'About Sass entry points'
+    href: '#about-sass-entry-points'
+  - text: 'Step 1: Install uswds-compile'
+    href: '#step-1-install-uswds-compile'
+  - text: 'Step 2: Create a gulpfile'
+    href: '#step-2-create-a-gulpfile'
+  - text: 'Step 3: Set USWDS version'
+    href: '#step-3-set-uswds-version'
+  - text: 'Step 4: Customize path settings'
+    href: '#step-4-customize-path-settings'
+  - text: 'Step 5: Export compile functions'
+    href: '#step-5-export-compile-functions'
+  - text: 'Step 6: Initialize your project'
+    href: '#step-6-initialize-your-project'
+  - text: 'Step 7: Verify successful installation'
+    href: '#step-7-verify-successful-installation'
 ---
 
 U.S. Web Design System source code is written in Sass, a powerful stylesheet language that builds automation, functions, and logic into CSS. Browsers can’t read native Sass files, so these files need to be compiled into CSS — usually a single CSS file — before we can use them. Basically, when you develop with the Design System, you do all your stylesheet work in Sass then use a compiler to convert that Sass into CSS.
 
-We’ve developed `uswds-compile`, a tool hosted on GitHub, to help teams copy USWDS static assets and transform USWDS Sass into browser-readable CSS. [Step 2](#step-2-install-uswds-compile) describes how to install this tool. When you use `uswds-compile`, we will apply Autoprefixer to all compiled code.
+## Introducing uswds-compile
+
+We’ve developed `uswds-compile`, a tool [hosted on GitHub](https://github.com/uswds/uswds-compile), to help teams compile USWDS Sass. This tool provides simple Gulp 4 functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
+
+We recommend using `uswds-compile` as a quick way to get up and running with USWDS Sass compilation.
```

Comment:
Not sure if this actually is the official recommended path for all users, but as a new user (especially one unfamiliar with Sass compilation) I felt like I would want to know if I should choose `uswds-compile`. An explicit recommendation felt like it would help guide a user make a choice.